src/opensignals/data/yahoo.py

```diff
@@ -107,7 +107,7 @@ def get_data(
         targets['friday_date'],
         format='%Y%m%d'
     )
-    targets['target_6d'] = targets['target']
+    targets['target_4d'] = targets['target_4d']
```

Comment:
This line is not needed anymore. Could you remove it?

Reply:
Oops!
Thanks for review~!